### PR TITLE
gzip data in all ci-operator config maps

### PR DESCRIPTION
**Pull request by droslean**
/cc @openshift/openshift-team-developer-productivity-test-platform @petr-muller 

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

**Comment by droslean** [comment]
follow-up of https://github.com/openshift/ci-tools/pull/1661

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -711,21 +711,25 @@ config_updater:
       clusters:
         app.ci:
         - ci
+      gzip: true
```
Comment (alvaroaleman) [inline core-services/prow/02_config/_plugins.yaml:714]: Can we just set `gzip: true` in the global config_updater config and disable it in the cases where we need it disabled (if any)?

Comment (droslean) [reply core-services/prow/02_config/_plugins.yaml:714]: I don't think that this is a good idea. All templates will break for sure and I don't know what else. I wonder is ci-operator can handle it. Let me check this first.

Comment (droslean) [reply core-services/prow/02_config/_plugins.yaml:714]: Yep.. templates will break. https://github.com/openshift/ci-tools/blob/master/cmd/ci-operator/main.go#L506
I will check what else can break one by one.

**Comment by droslean** [comment]
@alvaroaleman we should just take this approach. Enabling global gzip will break things like ci-operator templates (this can be changed) and I am worried about the cluster profiles (which are directly mounted to job yamls) image mirroring stuff etc...

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**